rmgpy/molecule/atomtype.py

```diff
@@ -621,7 +621,7 @@ def get_features(self):
 
 ATOMTYPES['X'].set_actions(increment_bond=['X'], decrement_bond=['X'], form_bond=['X'], break_bond=['X'], increment_radical=[], decrement_radical=[], increment_lone_pair=[], decrement_lone_pair=[])
 ATOMTYPES['Xv'].set_actions(increment_bond=[], decrement_bond=[], form_bond=['Xo'], break_bond=[], increment_radical=[], decrement_radical=[], increment_lone_pair=[], decrement_lone_pair=[])
```

Comment:
should you also set the increment_bond on `Xv` ?

Reply:
If the reaction recipe action is `CHANGE_BOND` by `+1` between atoms *1 and *2, then the atomtypes for *1 and *2 are updated based on the atomtypes in the `increment_bond` list for those atoms (and vice versa, `-1` change and `decrement_bond`).  We do not have to add `Xo` to `increment_bond` for `Xv` with the changes in this PR because when we apply the `CHANGE_BOND` action on an `Xv` atomtype, we first apply the `FORM_BOND` action, and since `Xo` is in the `form_bond` list for `Xv`, `Xv` is updated to `Xo`, and then the bond Is changed to the `Xo` atomtype.